db/migrate/20210629104700_populate_participant_profiles.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class PopulateParticipantProfiles < ActiveRecord::Migration[6.1]
+  class EarlyCareerTeacherProfile < ApplicationRecord
+  end
+
+  class MentorProfile < ApplicationRecord
+  end
+
+  class ParticipantProfile < ApplicationRecord
+    self.inheritance_column = nil
+  end
+
+  def up
+    MentorProfile.find_each do |ect_profile|
+      ParticipantProfile.create!(
+        id: ect_profile.id,
+        type: "ParticipantProfile::Mentor",
+        user_id: ect_profile.user_id,
+        school_id: ect_profile.school_id,
+        cohort_id: ect_profile.cohort_id,
+        core_induction_programme_id: ect_profile.core_induction_programme_id,
+      )
+    end
+
+    EarlyCareerTeacherProfile.find_each do |ect_profile|
```

Comment:
Are we likely to hit a race condition where production is still adding records to this table after you've copied them all across? It seems like this change might require downtime to ensure we don't lose data.